src/_components/card/card.css

```diff
@@ -289,3 +344,10 @@ a.card-mode:focus .card-inner {
   background: var(--color--black);
   color: var(--color--white);
 }
+
+.card-mode[data-breaking="true"] .card-inner,
+.card-mode[data-breaking="true"]:hover,
+[data-breaking="true"]:not([class*="bg--"]):not([class*="card-mode--article"]) {
```

Comment:
Vores selectors er ret vilde efterhånden.

Reply:
🏅

Reply:
Ja, jeg gav en medalje til mig selv for den flotte selector :P

Reply:
```
.༼•̫͡•༽ {
  background: GhostWhite;
  color: darken(GhostWhite, 10);
  font-family: 'Creepster', cursive;
}
```